wordpress-mobile/gutenberg-mobile
# Port InlineToolbar to Gutenberg as BlockMobileToolbar

Step 4 of https://github.com/wordpress-mobile/gutenberg-mobile/issues/958
This is the next step in our port of the BlockList to gutenberg.

Gutenberg PR https://github.com/WordPress/gutenberg/pull/16177

### Testing Instructions
- Boot the app on iOS and Android
- Check that the inline toolbar (toolbar that appears under a block when focused) is still functional meaning you can move a block up, down, remove it and access its settings in some cases.

Update release notes:
